weatherman.py

```diff
@@ -0,0 +1,166 @@
+"""Weatherman
+This module generates yearly reports on weather data collected from files taken
+from the user provided directory.
+Report no. 1 gives the max/min temperature and max/min humidity of each year.
+Report no. 2 gives the hottest day of each year and the temperature on that day.
+"""
+
+import argparse
+import os
+import sys
+
```

Comment:
Hamza, I've uploaded the updated file. I don't know who else to tag as a reviewer, as no one has approached me as yet. Do let me know.
Plus there is one continuous problem which I have already showed you, that max humidity and min humidity columns are swapped with each other, also I've highlighted another problem with these columns.
![swapped](https://user-images.githubusercontent.com/43273101/47644208-59e84280-db8f-11e8-9fd9-b49cdf897294.png)

Reply:
Saad has gotten in-touch. Adding him to the reviewers.